dashboard/src/app/workspaces/workspace-details/workspace-details.controller.ts

```diff
@@ -286,9 +286,13 @@ export class WorkspaceDetailsController {
    * @param {string} namespaceId
    * @return {string}
    */
-  getNamespaceLabel(namespaceId: string): string {
```

Comment:
lets keep namespace here for 2 reasons:
1. Che doesn't know about teams 
2. namespace can be also user name, organization and "team"